src/typ/hydrator.rs

```diff
@@ -28,15 +31,21 @@ pub struct ScopeResetData {
     created_type_variable_ids: im::HashSet<usize>,
 }
 
-impl Hydrator {
-    pub fn new() -> Self {
+impl Default for Hydrator {
+    fn default() -> Self {
```

Comment:
Why is this done?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default